stdlib/std.ncl

```diff
@@ -2326,7 +2603,8 @@
       = fun s => %num_from_str% s,
 
     to_bool
-      | BoolLiteral -> Bool
+      | String -> Bool
```

Comment:
Same here, was this meant to be a type instead of a contract?

Reply:
This one as well, yes.